modules/ocf_tv/files/i3/config

```diff
@@ -148,6 +153,32 @@ mode "resize" {
 
 bindsym $mod+r mode "resize"
 
+# Volume Control
+mode "volume" {
+    # Volume up
+    bindsym i exec --no-startup-id amixer -D pulse sset Master 5%+
+    bindsym Shift+i exec --no-startup-id amixer -D pulse sset Master 15%+
+    # Volume down
+    bindsym d exec --no-startup-id amixer -D pulse sset Master 5%-
+    bindsym Shift+d exec --no-startup-id amixer -D pulse sset Master 15%-
+    # Mute / Max
+    bindsym m exec --no-startup-id amixer -D pulse sset Master 0%
+    bindsym Shift+m exec --no-startup-id amixer -D pulse sset Master 100%
+    # back to normal: Enter or Escape
+    bindsym Return mode "default"
+    bindsym Escape mode "default"
+}
+
+bindsym $mod+Shift+v mode "volume"
+bindsym $mod+v exec --no-startup-id pavucontrol
+
+# htop
+bindsym $mod+i exec --no-startup-id $term -e htop
+
+# Web Browser
+bindsym $mod+w exec --no-startup-id google-chrome-stable
+bindsym $mod+Shift+w exec --no-startup-id $browseralt
```

Comment:
I'm not opposed to the "volume mode," (since it could make adjusting volume easy while on vncviewer) but are these shortcuts necessary when we have the launcher?

Reply:
They are much faster. Kinda the entire point of i3.